application/comit_node/Cargo.toml

```diff
@@ -80,10 +88,10 @@ lazy_static = "1"
 maplit = "1"
 memsocket = "0.1"
 pretty_env_logger = "0.2"
+serde_urlencoded = "0.5.4"
 spectral = "0.6"
-tiny-keccak = "1.4"
 testcontainers = "0.6"
-serde_urlencoded = "0.5.4"
+tiny-keccak = "1.4"
```

Comment:
wat

Reply:
Because I used `cargo add` to add a dependency it reformarted the `Cargo.toml` file.
Hopefully most dependencies are unchanged.